src/main/java/com/metamx/metrics/JvmThreadsMonitor.java

```diff
@@ -0,0 +1,58 @@
+package com.metamx.metrics;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableMap;
+import com.metamx.emitter.service.ServiceEmitter;
+import com.metamx.emitter.service.ServiceMetricEvent;
+import java.util.Map;
+import org.gridkit.lab.jvm.perfdata.JStatData;
+import org.gridkit.lab.jvm.perfdata.JStatData.LongCounter;
+
+public class JvmThreadsMonitor extends AbstractMonitor
```

Comment:
1. I think this is even possible that jstat mapped file will be removed altogether. In addition to the general trend of deprecating/removing non-standard things and replacing them with standards (see sun.misc.Unsafe in jdk9+), jstat design with mapped file is considered particularly poorly designed and a bone in the throat (see https://issues.apache.org/jira/plugins/servlet/mobile#issue/CASSANDRA-9242 for instance).